test/claim/claim-with-request.testSuite.ts

```diff
@@ -286,4 +289,80 @@ export const claimWithRequestTestSuite = () => {
       requestOrigin
     );
   });
+  it(`should fetch all issued claims by revoker did`, async () => {
+    const [requester, revoker, issuer] = await Promise.all([
+      randomUser(),
+      randomUser(),
+      randomUser(),
+    ]);
+    await createRole(
+      {
+        name: 'testrevoke',
+        issuerDid: [issuer.wallet.address],
+        revokerDid: [revoker.wallet.address],
+        ownerAddr: issuer.wallet.address,
+      },
+      roleService
+    );
+    const claimData = await createClaimRequest(
+      'testrevoke.roles.e2e.iam.ewc',
+      requester,
+      issuer,
+      201
+    );
+    await issueClaimRequest(claimData, issuer, 201);
+
+    const result = await claimService.getByRevoker({
+      currentUser: revoker.did,
+    });
+    expect(result).toHaveLength(1);
+  });
+  it(`should fetch all issued claims by revoker did with namespace filtered`, async () => {
+    const [requester, revoker, issuer] = await Promise.all([
+      randomUser(),
+      randomUser(),
+      randomUser(),
+    ]);
+    const roleOne = {
+      name: 'tstrevokens',
+      nameSpace: `roles.spaceOne.e2e.iam.ewc`,
+      issuerDid: [issuer.wallet.address],
+      revokerDid: [revoker.wallet.address],
+      ownerAddr: issuer.wallet.address,
+    };
+    const roleTwo = {
+      name: 'tstrevokens',
+      nameSpace: `roles.spaceTwo.e2e.iam.ewc`,
+      issuerDid: [issuer.wallet.address],
+      revokerDid: [revoker.wallet.address],
+      ownerAddr: issuer.wallet.address,
+    };
+    await Promise.all([
+      createRole(roleOne, roleService),
+      createRole(roleTwo, roleService),
+    ]);
+    const claimDataOne = await createClaimRequest(
+      'tstrevokens.roles.spaceOne.e2e.iam.ewc',
+      requester,
+      issuer,
+      201
+    );
+    const claimDataTwo = await createClaimRequest(
+      'tstrevokens.roles.spaceTwo.e2e.iam.ewc',
+      requester,
+      issuer,
+      201
+    );
+    await Promise.all([
+      issueClaimRequest(claimDataOne, issuer, 201),
+      issueClaimRequest(claimDataTwo, issuer, 201),
+    ]);
+    const result = await claimService.getByRevoker({
+      currentUser: revoker.did,
+      filters: {
+        namespace: 'tstrevokens.roles.spaceTwo.e2e.iam.ewc',
+      },
+    });
+    expect(result).toHaveLength(1);
```

Comment:
```suggestion
    const result = await claimService.getByRevoker({
      currentUser: revoker.did,
      filters: {
        namespace: 'tstrevokens.roles.spaceTwo.e2e.iam.ewc',
      },
    });
    expect(result).toHaveLength(1);
```
also http call, and we should check what claims it returns, checking only length is not enough